sass/utils/_mixins.scss

```diff
@@ -40,3 +40,14 @@
         max-height: $line-height * $lines * 1px;
     }
 }
+
+@mixin linear-gradient($top, $bottom){
```

Comment:
0/5 on whether we keep this, but we don't really need this mixin since every browser we support also supports linear-gradient according to [this](https://caniuse.com/#feat=mdn-css_types_image_gradient_linear-gradient)

Reply:
Cool, I can just use it as it is and remove it.